python/apps/taiga/src/taiga/workflows/api/__init__.py

```diff
@@ -120,3 +108,53 @@ async def create_workflow_status(
         color=form.color,
         workflow=workflow,
     )
+
+
+################################################
+# update workflow status
+################################################
+
+
+@routes.stories.patch(
+    "/projects/{project_id}/workflows/{workflow_slug}/statuses/{status_slug}",
+    name="project.workflow.status.patch",
+    summary="Update workflow status",
+    responses=WORKFLOW_STATUS_200 | ERROR_404 | ERROR_422 | ERROR_403,
+)
+async def update_workflow_status(
+    request: AuthRequest,
+    form: UpdateWorkflowStatusValidator,
+    project_id: B64UUID = Query(None, description="the project id (B64UUID)"),
+    workflow_slug: str = Query(None, description="the workflow slug (str)"),
+    status_slug: str = Query(None, description="the status slug (str)"),
+) -> WorkflowStatusSerializer:
+    """
+    Updates a workflow status in the given project workflow
+    """
+    workflow = await get_workflow_or_404(project_id=project_id, workflow_slug=workflow_slug)
```

Comment:
for a workflow status, this should be "workflow_status = await get_workflow_status_or_404" because it's the resource we want to patch, shouln't be?

Reply:
I get what you mean:
- extract the project or the workflow to check permissions (you chose the latter)
- extract the status to proper find the resource

Oks, we can let it be that way :)